pkg/cmd/create.go

```diff
@@ -152,6 +152,11 @@ func GetCreateOptions(ctx context.Context, clusterStackPath string) (*CreateOpti
 				return nil, fmt.Errorf("failed to download release asset: %w", err)
 			}
 
+			createOption.LatestReleaseHash, err = hash.ParseReleaseHash("./.tmp/release/hashes.json")
```

Comment:
@janiskemper what happens if two csmctl commands run in parallel? Is it possible that one that there is a conflict?

Reply:
yes there is. Can you open an issue for this? That goes beyond the scope of this PR.

Reply:
@janiskemper here the new issue: https://github.com/SovereignCloudStack/csctl/issues/145